src/bootstrap/src/core/build_steps/doc.rs

```diff
@@ -1194,6 +1197,26 @@ impl Step for RustcBook {
         builder.run(&mut cmd);
         drop(doc_generator_guard);
 
+        // Run target-docs generator
+        let mut cmd = builder.tool_cmd(Tool::TargetDocs);
+        cmd.arg(builder.src.join("src/doc/rustc/target_infos"));
+        cmd.arg(&out_src_listing);
+        cmd.env("RUSTC", &rustc);
+        // For now, we just check that the files are correct but do not generate output.
+        // Let the user override it to TARGET_CHECK_ONLY=0 for testing, but use 1 by default.
+        // See #120745 for more info.
```

Comment:
Can you put the actual link here? It's not clear that whether this is an issue or a PR id.

Reply:
Issues and PRs share a namespace :)

Reply:
Interesting, I didn't know that. I would still prefer the actual link though, since most editors will be able to open it by clicking on it.